sql/mysql/src/TMySQLStatement.cxx

```diff
@@ -758,17 +775,18 @@ Bool_t TMySQLStatement::SetSQLParamType(Int_t npar, int sqltype, Bool_t sig, ULo
       case MYSQL_TYPE_DATE:
       case MYSQL_TYPE_TIMESTAMP:
       case MYSQL_TYPE_DATETIME: allocsize = sizeof(MYSQL_TIME); doreset = true; break;
-      default: SetError(-1,"Nonsupported SQL type","SetSQLParamType"); return kFALSE;
+      default: SetError(-1,"None-supported SQL type","SetSQLParamType"); return kFALSE;
    }
 
-   if (allocsize > fgAllocSizeLimit) allocsize = fgAllocSizeLimit;
+   if ((fgAllocSizeLimit > 256) && (allocsize > fgAllocSizeLimit))
```

Comment:
Could add a comment (or use a constant) explaining the semantic of the 256 here?

Reply:
Values over 256 bytes used for custom data (like blob or large string). 
If user changes `fgAllocSizeLimit` value to wrongly, this will affect only custom user data but not normal data types